src/server/common.ts

```diff
@@ -44,13 +44,18 @@ export interface DevErrorResult {
     status: 500;
     results: ErrorObject[];
 }
+export interface DuplicateErrorResult {
+    status: 409;
```

Comment:
Instead of making a new type, we could change UserErrorResult to have:
```
status: 400|409
```

The subtype doesn't really matter that much, because the shape & info and how it's handled is the same between the two statuses